lib/mayaUsd/ufe/UsdUndoInsertChildCommand.cpp

```diff
@@ -124,16 +124,20 @@ bool UsdUndoInsertChildCommand::insertChildRedo()
     bool status = SdfCopySpec(_childLayer, _usdSrcPath, _parentLayer, _usdDstPath);
     if (status)
     {
+        // we shouldn't rely on UsdSceneItem to access the UsdPrim since 
+        // it could be stale. Instead we should get the USDPrim from the Ufe::Path
+        const auto& desPrimUsd = ufePathToPrim(_ufeDstPath);
```

Comment:
Same a line 164: this will give the wrong stage at line 134.

Reply:
Naming nit-pick: should use "dst" or "des" prefix for "destination", but not both.